hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -960,18 +957,12 @@ protected boolean evictInternal(Data keyData) {
     public void evictAll() {
         ClientMessage request = MapEvictAllCodec.encodeRequest(name);
         invoke(request);
-
-        clearNearCachesOnLiteMembers();
     }
 
     @Override
     public void loadAll(boolean replaceExistingValues) {
         ClientMessage request = MapLoadAllCodec.encodeRequest(name, replaceExistingValues);
         invoke(request);
-
-        if (replaceExistingValues) {
```

Comment:
Could not find a test for this, so I wrote one: https://github.com/hazelcast/hazelcast/pull/8869/files#diff-ec815c78e843f777f47642c555a0d9b2R140

:+1: